### DISCUSSION TITLE
ideas for an article

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#2826] [General]
The first attempt on getting an article about Kiosk published did not work out.
Here we collect ideas for Kiosk topics about which one could write an article.

**Comment by urapadmin** [comment]
@lbestock 
- longevity of digital repositories and how Kiosk makes sure your data survives it

**Comment by luizaogs** [comment]
(I saw this because you added the discussion with release notes. If it is unwanted input, then feel free to ignore.)

Laurel and I talked about this right after the article was rejected (I guess a year ago now). The next step in that conversation was to bring you into it, Lutz, but we never got there.

If the primary goal is still for archaeologists to read about Kiosk and potentially decide to adopt it, I think the main priority should be to publish in a venue that they actually read. I'm not sure if the topic suggested above would be accepted in any such venue (well, other than maybe Advances in Archaeological Practice, but we already tried that). If that is still the main goal, my suggestion had been to look for successful articles that discuss digital recording and where they were published. I did some initial research and there are a few examples in Journal of Field Archaeology -- those are in the Google drive. My suggestion would be to follow that model and publish Kiosk primarily as it was used in Uronarti, publishing some Uronarti data at the same time but making the focus on how the recording has been used and why it was useful. Now that BUAP is also using Kiosk, it could in fact be interesting to use both Uronarti and Abydos as case studies, publishing a bit of actual data from both and fronting the use of Kiosk in both field and legacy data recording. And then, since the whole deal with Kiosk is its universality, there could be a section discussing its flexibility in terms of how it has been used for other projects, with some specifics but obviously no actual project data.

Anyway, just a suggestion.

**Reply by urapadmin** [reply]
The main goal from my point of view is to make Kiosk referenceable (is that a word?). In academia nothing exists that isn't referenced. So I thought we need to get an article out so that our users and people who publish their own articles can reference Kiosk. That way we'd enter the indefinite, self-perpetuating realm of academic discourse. 

But ja, I think your way is perfectly doable. And the question about archiving and longevity might even be one of the paragraphs, e.g. when BUAP is the topic. I must admit that my own energy to get involved in publishing about this is very low. There is so much else to do that is more up my alley.

**Reply by luizaogs** [reply]
I'm still happy to help with this, and I do think some stuff from the previous attempt could be reused (particularly for a potential section on other projects that use Kiosk).

**Reply by lbestock** [reply]
I am hoping I will have the energy to return to this sometime this December or thereabouts. I agree the focus on what has been done in the field and with legacy data is a good one; I think publishing data is very secondary, though, and that methods should be central. Though we might use from BUAP the Middle Kingdom material. There is not much of it and precisely because it was not the focus of the excavations and was overshadowed by the Early Dynastic and Late Roman stuff, it will be interesting to me to see if something can come from it that is aided by having digitized in a structured way. @luizaogs I will want your help. IF there is any additional Abydos stuff to bring in that will also be an interesting point of comparison. The BUAP stuff is particularly cool because there was a change from unstructured to structured(ish) recording on paper. I'm very curious what's going to happen now that the students are starting to try to turn the unstructured into structure. The ANC context recording stuff from NYU is all just the biggest pile of burger ahh personal observations imaginable. BUT they already put a lot of thought into digitizing, and that is useful for us as a contrast. A photodatabase that allowed the inputting of dates in - not kidding - four different formats so that in the end nothing could be sorted by dates and where context information is variably in the filename, the description, or absent entirely, shows very neatly why digitizing alone is not the answer.

**Reply by luizaogs** [reply]
Data to illustrate methods is what I mean. One of our issues with the previous attempt was that the discussion was too abstract. To counter that and to show what the advantages of Kiosk actually are we will need nuggets of actual data from both URAP and BUAP (when you get back to it you should read the JFA examples I pulled to see what I mean).

**Comment by lbestock** [comment]
One thing definitely needing to go in here: when Kiosk is used originally in the field it prevents double-assigning a number. This is an actual real problem in BUAP. There are double assigned locus numbers. What do we CALL the second iteration? Calling it 15B was a mistake - then that's not a number any more. Dropping the 15 is a problem - it is going to be 15 in the narrative, on plans, perhaps on tags and CM. That there IS NO GOOD SOLUTION in tackling this with legacy data is precisely why Kiosk should be used in the first place.

**Comment by luizaogs** [comment]
Laurel mentioned at dinner in Boston that having some sort of outline would help her get started on this, so there is now a draft outline in the Kiosk Google drive. I re-read the Journal of Field Archaeology articles I had mentioned and used them, as well as the discussion above, to try and come up with a structure that I thought would make sense.
